.github/workflows/preview-deployment.yml

```
@@ -102,6 +102,20 @@ jobs:
           eas-cache: true
           patch-watchers: true # Prevents ENOSPC errors on Ubuntu runners
 
+      - name: Prepare iOS build folder
+        run: |
+          # Generate the iOS folder with the Notification Service Extension
+          yarn expo prebuild --platform ios --clean
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)